assets.go

```diff
@@ -2,8 +2,11 @@ package achgateway
 
 import "embed"
 
-//go:embed migrations/*.sql
-var MigrationFS embed.FS
+//go:embed migrations/*mysql*.sql
+var MySqlMigrationFS embed.FS
+
+//go:embed migrations/*spanner*.sql
+var SpannerMigrationFS embed.FS
```

Comment:
Do files need to be named with the db towards the end? e.g. `003_cron_triggers.up.spanner.sql`

Reply:
Yeah I like that better. Does the migrations library not expect them to end in `.up.sql` though?

I guess if you guys are doing this internally already then it should be fine. The alternative would be `003_cron_triggers.spanner.up.sql`. 

I'll change these to your suggestion and verify it works.